torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -246,6 +246,10 @@ std::shared_ptr<SugaredValue> CUDAPythonModuleValue::attr(
   }
 
   py::object member = getattr(loc, field);
+  if (field == "Stream" || field == "Event") {
+    auto class_type = getCustomClass("__torch__.torch.classes.cuda." + field);
+    return std::make_shared<ClassValue>(class_type);
+  }
```

Comment:
This should moved above the `getattr` above.

Reply:
Done